### PR TITLE
Basic script based build for polygon-etl and changed to allow it to run.

### PR DESCRIPTION
**Pull request by Rob-Petri**
Basic script based build and deploy
    This will be replace with a ci/cd pipeline ASAP

Fix hardcoded dataset name in load dag

Fix upload_to_gcs to use newer api
    objects().insert is deprecated.  Changed to use new interface supported
    by the gcs library available in airflow.

Fixed args to render_template for newer lib
    The first arg is now omitted in the version of the lib available in airflow

### CODE REVIEW

File: build_scripts/build.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+
+source cur_env.sh > /dev/null
+
+[ -e "${DAG_DIR}" ] && rm -rf "${DAG_DIR}"
+[ -e "${BUILD_DIR}/airflow_vars.json" ] && rm -rf "${DAG_DIR}"
```
Comment (DefinitiveStephen) [inline build_scripts/build.sh:6]: nit: `rm -rf ${BUILD_DIR}/airflow_vars.json` ? it looks like `airflow_vars.json` technically is getting created under ${BUILD_DIR}, not under `${DAG_DIR}`

**Review by DefinitiveStephen — APPROVED** [review]
Looks good!

File: build_scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Building
+1. cd to the `build_scripts` directory
+2. Make sure cur_env.sh is a symlink to a good config file and verify all values
+3. run `build.sh`
+   - This will create a `build` directory which will contain
+     - `build/airflow_vars.json` Vars to import into the airflow web console
+     - `build/dags` The dags and support files
+4. Set the correct gcloud project. For example `gcloud config set project prj-x-demo-7c38`
+4. Create the working bucket by running `create_bucket.sh` only required once.
```
Comment (DefinitiveStephen) [inline build_scripts/README.md:22]: Note that this bucket may already exist if you have followed the steps outlined in the `composer-etl` scripts.

File: build_scripts/build.sh (hunk 1 above)
Comment (Rob-Petri) [reply build_scripts/build.sh:6]: Typo, good catch.  Thanks

**Pull request merged**